Explorer/Assets/DCL/Multiplayer/Connections/Archipelago/Rooms/IRealmRoomStrategy.cs

```diff
@@ -0,0 +1,9 @@
+﻿using DCL.Multiplayer.Connections.Rooms.Connective;
+
+namespace DCL.Multiplayer.Connections.Archipelago.Rooms
+{
+    internal interface IRealmRoomStrategy
+    {
+        ConnectiveRoom ConnectiveRoom { get; }
```

Comment:
Why did you decided to bring hard coupling here? It's impossible to replace the implementation for tests for example

Reply:
I directly followed the existing code: it was not requiring any abstractions so I didn't introduce them: of course I can replace it with an interface.

Reply:
I replaced by the interface